angularjs/manageVariable/edit.controller.js

```diff
@@ -267,7 +267,10 @@
 
             this.isUpdating = true;
 
-            tagManagerVariableModel.createOrUpdateVariable(this.variable, 'TagManager.addContainerVariable').then(function (response) {
+            var tempVariable = JSON.parse(JSON.stringify(this.variable));
```

Comment:
instead of JSON parse/stringify you could also simply use `angular.copy`

Reply:
thanks, didn't know about that. Done